code/modules/mining/remote_fob/remote_fob_computer.dm

```diff
@@ -50,13 +51,16 @@
 
 /obj/machinery/computer/camera_advanced/remote_fob/give_eye_control(mob/user)
 	. = ..()
+	icon_state = "fobpc-transfer"
 	user.lighting_alpha = 120
 	eyeobj.name = "Remote Construction Drone"
 	eyeobj.register_facedir_signals(user)
 	if(eyeobj.eye_initialized)
 		eyeobj.setLoc(get_turf(spawn_spot))
 	
 /obj/machinery/computer/camera_advanced/remote_fob/attack_hand(mob/living/user)
+	if(machine_stat & (NOPOWER|BROKEN))
```

Comment:
You should call parent here i believe if you check they do a lot of checks like if the user is concious and if the machine is operational etc.

Look into the parents.

Reply:
they dont actually, i experimented with that before. it's just a long line of calling parent, all the way up to atom/movable which obviously doesnt have these kind of checks. i hadnt even noticed you could use this when broken/depowered before experimenting with the new sprites tbh.